pandas/tests/series/test_timeseries.py

```diff
@@ -422,6 +422,38 @@ def test_pct_change_shift_over_nas(self):
         expected = Series([np.nan, 0.5, 0.0, 2.5 / 1.5 - 1, 0.2])
         assert_series_equal(chg, expected)
 
+    @pytest.mark.parametrize(
+        "offset_1, offset_2",
+        product(
+            ["D", "W", "M"],
+            [
+                "B",
+                "C",
+                "W",
+                "M",
+                "SM",
+                "BM",
+                "CBM",
+                "MS",
+                "SMS",
+                "BMS",
+                "CBMS",
+                "Q",
+                "BQ",
+                "BA",
+                "A",
+                "AS",
+            ],
+        ),
+    )
+    def test_pct_change_anchored_freq(self, offset_1, offset_2):
+        s = Series(
```

Comment:
Can you add the issue number as a comment here?

Reply:
OK, it slipped my mind D:. I'll add it